recipes-browser/wpewebkit/wpewebkit_2.28.0.bb

```diff
@@ -0,0 +1,13 @@
+require wpewebkit.inc
+
+SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz"
+
+SRC_URI[sha256sum] = "a85cd3cb46206a4929a9562d53379a7e7e2ec1a3224b34e2dcf5da30bb906722"
+
+DEPENDS += " libwpe"
+RCONFLICTS_${PN} = "libwpe (< 1.2) wpebackend-fdo (< 1.2)"
```

Comment:
Should be update the references? <- @aperezdc

Reply:
I suspect since this is an `RCONFLICTS` its still correct. It really says "You can't install wpewebkit at the same time as anything earlier libwpe 1.2. It can be installed at the same time as 1.2 or later though"